src/model/index.ts

```diff
@@ -375,6 +404,26 @@ class Model implements WorkbookModel {
     this.transactionsBySecurityId_.set(value.security_id, txns);
   }
 
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  private TX_EQUITY_COMPENSATION(value: any) {
+    if (
+      value.object_type === "TX_EQUITY_COMPENSATION_ISSUANCE" &&
+      value.stock_class_id !== undefined
+    ) {
+      const key = `${value.stakeholder_id}/${value.stock_class_id}`;
+      const ids =
+        this.issuedSecuritiesByStakeholderAndNonPlanStockIds_.get(key) ||
+        new Set();
+      ids.add(value.security_id);
+      this.issuedSecuritiesByStakeholderAndNonPlanStockIds_.set(key, ids);
+      this.nonPlanStockIds.add(value.stock_class_id);
```

Comment:
Similar to what we have with warrants, we want to keep track of the common stock class id related to a `TX_EQUITY_COMPENSATION_ISSUANCE`